docs/connection-and-channel-management.md

```diff
@@ -0,0 +1,53 @@
+# Connection and channel management
+
+Spaces SDK uses the Ably SDK client [connections](https://ably.com/docs/connect?lang=javascript) and [channels](https://ably.com/docs/channels?lang=javascript) to provide higher level features like cursors or locations. Both connections and channels will transition through multiple states throughout their lifecycle. Most state transitions (like a short loss in connectivity) will be handled by the Ably SDK, but there will be use cases where developers will need to observe these states and handle them accordingly.
+
+There is no dedicated Spaces API to handle the connection or channels, however the relevant Ably SDK APIs are available. This document describes how to access a connection and channels on Spaces, and where to find information about how to handle their state changes.
+
+## Connection
+
+- [Connections](https://ably.com/docs/connect?lang=javascript)
+- [Connection state and recovery](https://ably.com/docs/connect/states?lang=javascript)
+
+Each `Spaces` instance requires a connection. When initializing, an ably client is passed as an argument. A client exposes a `connection` object, which is an Event emitter:
```

Comment:
Minor suggestion, but instead of "Each `Spaces` instance requires a connection", I think "Each `Spaces` instance establishes a connection to Ably using the Core SDK" is more accurate.  

Proposed change:

When initializing the Spaces SDK, an *Ably* client is passed as a required argument. The Spaces client exposes the underlying `connection` which is an EventEmitter: